classes/imx-boot-container.bbclass

```diff
@@ -29,7 +29,7 @@ ATF_MACHINE_NAME:append = "${@bb.utils.contains('MACHINE_FEATURES', 'optee', '-o
 # appropriate dependencies for populate binaries task
 do_resolve_and_populate_binaries[depends] += " \
     ${@' '.join('%s:do_deploy' % r for r in '${IMX_EXTRA_FIRMWARE}'.split() )} \
-    imx-atf:do_deploy \
+    ${DEFAULT_ATF_PROVIDER}:do_deploy \
```

Comment:
I'd prefer if this variable had the IMX_ prefix. It shows to user it is something i.MX specific.

Reply:
Make sense, would change this in _V2_.